TF_Settings_Web/src/Pages/Visuals/VisualsScreen.tsx

```diff
@@ -91,6 +94,13 @@ const VisualsScreen: React.FC = () => {
 
     if (!isDesktop() || !hasReadConfig) return <></>;
 
+    window.onpointerup = () => {
```

Comment:
:eyes: These should be set in a useEffect and we should register and unregister as window is global